src/cm_colors/core/optimisation.py

```
@@ -360,28 +358,34 @@ def check_and_fix_contrast(
             }
 
     accessible_text = generate_accessible_color(text, bg, large)
-    wcag_level = get_wcag_level(accessible_text, bg)
+    final_contrast = calculate_contrast_ratio(accessible_text, bg)
+                
+    aa_target = 3.0 if large else 4.5
+
+    is_wcag_aa = final_contrast >= aa_target
```

Ruff (0.14.5)

Local variable `is_wcag_aa` is assigned to but never used

Remove assignment to unused variable `is_wcag_aa`

(F841)